starlite/response.py

```diff
@@ -74,6 +75,10 @@ def serializer(value: Any) -> Dict[str, Any]:
         """
         if isinstance(value, BaseModel):
             return value.dict()
+        if isinstance(value, SecretStr):
```

Comment:
@cofin is this ordering correct?
wouldn't you want the SecretStr to take priority over all other types?

Reply:
why?